modules/obam/manifests/params.pp

```diff
@@ -17,6 +17,11 @@
 class obam::params inherits ob_common::params{
 
   $start_script_template  = 'bin/wso2server.sh'
+  
+  #File paths obam
```

Comment:
Will indicate in the comment that the specified file paths are replaceable for different specification